lib/signs_ui_web/templates/signs/index.html.eex

```diff
@@ -1,8 +1,12 @@
 <h3>Countdown Signs</h3>
 <div>
-  <ul>
+  <%= form_for @conn, signs_path(@conn, :update), [as: :signs], fn f -> %>
     <%= for {sign_id, sign} <- @signs do %>
-    <li><%= sign_id%> : <%= if sign.enabled?, do: "Enabled", else: "Disabled" %></li>
+      <div>
+        <label for=<%= sign_id %>><%= sign_id %></label>
```

Comment:
Little thing, but can you actually switch the order here, so all the checkboxes line up?

Reply:
I was skeptical but that looks much better